### ISSUE TITLE
No Binding to Web App Created

### ISSUE THREAD

**Issue by stevendelapena** [ohadschn/letsencrypt-webapp-renewer#12]
I followed the install instructions and everything worked well except the binding for the SSL was not added to the web app. Once I added it manually, the SSL worked fine.

Is there something missing in the instructions so this binding can be automated?

**Comment by ohadschn** [comment]
Can you share the WebJob logs?

Generally speaking there's nothing special you should do, though I noticed it might take a few minutes and you need to refresh the Azure Portal window to see the binding. Could it be that it was successful but you were just looking at a stale dashboard (or didn't wait a few minutes for it to kick in)?

**Comment by stevendelapena** [comment]
I saw the web job finished, then looked at the web app and saw the cert was added, but not the binding. I assumed that if both were done, that the binding would've been there.

I just deleted the cert and binding and tried again. I got the same result - but forced a refresh of the browser and it showed. Looks like an Azure portal issue not picking up the change with their js client app.

Thanks for confirming it should (and does) create the binding.
